frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1415,6 +1415,88 @@
             }
         };
 
+        vm.challengePhaseDialog = function(ev, phase) {
+            vm.page.challenge_phase = phase;
+            vm.phase_start_date = new Date(phase.start_date);
```

Comment:
Can we please use `CamelCase` in the variable names.

Reply:
Done!

Reply:
@RishabhJain2018 Done!